iFixit/react-commerce
# Font load optimization

closes #559 

This PR moves font links (and other link/meta) from `next/head` to `_document.tsx`.
This enables [nextjs font optimization](https://nextjs.org/docs/basic-features/font-optimization).

## QA

1. Visit Vercel preview
2. Verify that the font is not only linked, but that css `@font-face` instruction have been embedded in the page at build time
